specification/migrate/resource-manager/Microsoft.Migrate/stable/2019-05-01/migrate.json

```diff
@@ -0,0 +1,3748 @@
+          "readOnly": true
+        },
+        "updatedTimestamp": {
+          "type": "string",
+          "readOnly": true
+        },
+        "agentProperties": {
+          "$ref": "#/definitions/CollectorAgentProperties"
+        }
+      }
+    },
+    "CollectorAgentProperties": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true
+        },
+        "version": {
+          "type": "string",
+          "readOnly": true
+        },
+        "lastHeartbeat": {
+          "type": "string",
+          "format": "date-time",
+          "x-nullable": true,
+          "readOnly": true
+        },
+        "spnDetails": {
+          "$ref": "#/definitions/CollectorBodyAgentSpnProperties"
+        }
+      }
+    },
+    "VMwareCollector": {
+      "type": "object",
+      "properties": {
+        "eTag": {
+          "type": "string"
+        },
+        "id": {
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "$ref": "#/definitions/CollectorProperties"
+        }
+      }
+    },
+    "HyperVCollectorList": {
+      "type": "object",
+      "description": "List of Hyper-V collectors.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of Hyper-V collectors.",
+          "items": {
+            "$ref": "#/definitions/HyperVCollector"
+          }
+        }
+      }
+    },
+    "VMwareCollectorList": {
+      "type": "object",
+      "description": "List of VMware collectors.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of VMware collectors.",
+          "items": {
+            "$ref": "#/definitions/VMwareCollector"
+          }
+        }
+      }
+    },
+    "VmFamily": {
+      "type": "object",
+      "description": "VM family name, the list of targeted azure locations and the category of the family.",
+      "properties": {
+        "familyName": {
+          "type": "string",
+          "description": "Name of the VM family.",
+          "readOnly": true
+        },
+        "targetLocations": {
+          "type": "array",
+          "description": "List of Azure regions.",
+          "readOnly": true,
+          "items": {
+            "type": "string",
+            "description": "Azure region name.",
+            "readOnly": true
+          }
+        },
+        "category": {
+          "type": "array",
+          "description": "Category of the VM family.",
+          "readOnly": true,
+          "items": {
+            "type": "string",
+            "description": "VM family Category.",
+            "readOnly": true
+          }
+        }
+      }
+    },
+    "AssessmentOptionsResultList": {
+      "type": "object",

```

Comment:
Write 404 in the responses may casue some errors for SDK generation process. I may need to check the details again and let you know the conclusion.

Reply:
@shsingla Hello. Usually, we will just use default as the error response. Is there any specific reason you need to use 404?

Reply:
Done moved to default.